### PR TITLE
Some improvements in submit form

### PR DESCRIPTION
**Pull request by mdsnins**
`index.html` now supports these features

- Check whether there are empty inputs or not.
- Automatically trim whitespaces in 3 input fields, `Student ID`, `Name`, `Token`
- Test a simple regex pattern, `/20[0-9]{6}/` to Student ID, which may prevent a typo in `Student ID`
- When the uploaded file's name is not `myfunc.fs` (case insensitive), confirm it to the user.

### CODE REVIEW

File: src/Server/index.html [hunk 1]
```diff
@@ -1,40 +1,75 @@
...
+    <script src="http://code.jquery.com/jquery-latest.js"></script>
+    <script>
+        function _trim(obj) {
+            obj.val(obj.val().trim());
+        }
+        function _validate() {
+            let _sid = $("#sid").val();
+            let _code = $("#code").val().split("\\").pop();
+
+            if(!/20[0-9]{6}/.test(_sid)) {
```
Comment (sangkilc) [inline src/Server/index.html]: I would be happier if you can relax the condition to just check eight digits: no need to check if the number starts with `20`.

File: src/Server/index.html [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+                alert("Wrong student id");
+                return -1;
+            }
+            if(_code.toLowerCase !== "myfunc.fs") {
+                if(confirm(`You're trying to submit ${_code}\nYou might submit myfunc.fs\nIs it sure?`)) {
```
Comment (sangkilc) [inline src/Server/index.html]: You might submit ... Is it sure?
->
Are you sure?

**Review by sangkilc — CHANGES_REQUESTED** [review]
Thanks for your contribution. Can you fix the two things?

**Comment by mdsnins** [comment]
I changed both of them!
Would you check once more?

**Comment by sangkilc** [comment]
Looks good to me. Your name please!

**Comment by mdsnins** [comment]
This is 20170171 Jihwan Kim, SoC.
Thanks!

**Pull request merged**